x/did/types/did.go

```diff
@@ -234,27 +227,179 @@ func (doc DIDDocument) VerificationMethodFrom(relationships []*VerificationRelat
 	return VerificationMethod{}, false
 }
 
-func ValidContexts(ctxs []string) bool {
-	if len(ctxs) == 0 || ctxs[0] != ContextDIDV1 { // the 1st one must be ContextDIDV1
+type Contexts []Context
```

Comment:
From here, contexts hand implementation of the part.

Reply:
Let me also test this manually. I'm not sure if this implementation works well because it's a bit complicated.

Reply:
I tested that it works well. But, what do you think about making it simpler like this: https://github.com/medibloc/panacea-core/commit/89cc7637c09319e51c700b2efe70b4ba9dd21501.
I referred to the following guides:
- https://github.com/gogo/protobuf/blob/master/custom_types.md
- https://github.com/gogo/protobuf/blob/master/test/t.go

Reply:
@youngjoon-lee 
Thank you.
It has been modified more simply.

https://github.com/medibloc/panacea-core/pull/129/commits/1ccd1847596973556f4908f9e000a9d7b05795b6